include/threads.php

```diff
@@ -251,7 +251,7 @@ function delete_thread($itemid, $itemuri = "") {
 	}
 
 	// Using dba::delete at this time could delete the associated item entries
```

Comment:
I love when a comment just answers a question I was about to write!

Reply:
This is the reason why I write such comments. They also serve as a reminder to myself.